gammapy/irf/effective_area_table.py

```diff
@@ -1,14 +1,19 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import print_function, division
 import numpy as np
+import matplotlib.pyplot as plt
```

Comment:
Same here, matplotlib is an optional dependency ... people should be able to run analyses without it.
So move this import into the methods where you use it – yes, this is duplicating the import a few times, there's not better way.